Html/running.html

```diff
@@ -176,5 +114,8 @@ <h1>Types of race 5k-10k-21k-42k</h1>
       </tbody>
      </table>
     </div>
+  </div><br><br>
```

Comment:
please remove all the <br> you use to separate sections and elements, you can achieve the same exact thing by using margin or padding using css. and the code looks cleaner

Reply:
Ok Jose, thank you!